test/authenticator.spec.ts

```diff
@@ -248,20 +228,18 @@ describe('Decentraland Crypto', function () {
       const result = await Authenticator.validateSignature(
         'QmUe3LmUJ4NACAKJzwQhn5rZVpLLSyBLWBmTSzJYEesDNx',
         chain,
-        new HttpProvider(
-          'https://mainnet.infura.io/v3/640777fe168f4b0091c93726b4f0463a'
-        )
+        new HTTPProvider('https://rpc.decentraland.org/mainnet')
```

Comment:
Can we use https://rpc.decentraland.org/mainnet?project=dcl-crypto-test so we can discriminate the usage?